features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml

```diff
@@ -120,6 +120,10 @@
 			<Parameter name="YahooOAuthzEndpoint">https://api.login.yahoo.com/oauth2/request_auth</Parameter>
 			<Parameter name="YahooUserInfoEndpoint">https://social.yahooapis.com/v1/user/</Parameter>
 		</AuthenticatorConfig>
+		<AuthenticatorConfig name="MobileConnectAuthenticator" enabled="true">
+			<Parameter name="MobileConnectKey">3689e589-419e-4676-8130-ec15a4f898f0</Parameter>
```

Comment:
Since these are random sandbox credentials shall we simply have 'mobileConnectClientIdValue' and 'mobileConnectClientSecret' as the values?